packages/firestore/test/unit/specs/offline_spec.test.ts

```diff
@@ -83,6 +83,7 @@ describeSpec('Offline:', [], () => {
           // If the next (already scheduled) connection attempt fails, we'll move
           // to unknown since there are no listeners, and stop trying to connect.
           .watchStreamCloses(Code.UNAVAILABLE)
+          .runTimer(TimerId.ListenStreamIdle)
```

Comment:
You've "fixed" the test to accommodate your changes, but the test name and comments need to be updated too as it's wrong/misleading as-is.  In fact, the test you added below has a title directly contrary to this one.